### PR TITLE
chore(node-project): adds `gh` cli rebuild example to doc string

### PR DESCRIPTION
**Pull request by pgollucci**

### CODE REVIEW

**Review by pgollucci — COMMENTED** [review]
@projen rebuild

**Comment by pgollucci** [comment]
Live exercise of comment above

File: src/node-project.ts [hunk 1]
```diff
@@ -470,6 +470,7 @@ export interface NodeProjectCommonOptions extends ProjectOptions {
   /**
    * The pull request bot command to use in order to trigger a rebuild and
    * commit of the contents of the branch. The command must be prefixed by "@projen", e.g. "@projen rebuild"
+   * `gh pr review $pr --comment -b "@projen rebuild"`
```
Comment (eladb) [inline src/node-project.ts:473]: Maybe we we can add a task to `NodeProject` that will do that:

```ts
project.addTask('pr:rebuild', { exec: 'gh pr review --comment ... ' });
```

**Review by eladb — APPROVED** [review]

File: src/node-project.ts (hunk 1 above)
Comment (pgollucci) [reply src/node-project.ts:473]: Sure.

Comment (pgollucci) [reply src/node-project.ts:473]: That likely depends on #356 for $pr unless we grab it from .git/ if someone does gh pr checkout

**Review by pgollucci — COMMENTED** [review]
@projen rebuild

**Pull request merged**